src/client/App.jsx

```diff
@@ -1,30 +1,94 @@
-import { useEffect, useState } from 'react';
-import './App.css';
-import MovieForm from './components/MovieForm';
-import UserForm from './components/UserForm';
+import { useEffect, useState } from "react";
+import "./App.css";
+import MovieForm from "./components/MovieForm";
+import UserForm from "./components/UserForm";
 
-const apiUrl = 'http://localhost:4000';
+const apiUrl = "http://localhost:4000";
```

Comment:
I believe Jules made a change for this - but just a caution - it's always a better idea to have apiUrls as an environment variable.